src/config.js

```diff
@@ -109,6 +109,10 @@ module.exports.setConfig = function setConfig(configData) {
         config.token = configData.token;
     }
 
+    if (configData.isEpsagonDisabled) {
```

Comment:
why not just `disable`?

Reply:
`isEpsagonDisabled` is already defined but not working